app/driver/settings.py

```diff
@@ -351,3 +351,6 @@
         'rest_framework.authentication.SessionAuthentication',
         'rest_framework.authentication.TokenAuthentication',
     )
+
+# These fields will be visible to read-only users
+READ_ONLY_FIELDS = ['Accident Details']
```

Comment:
Do we not want this filtering to apply to details fields generally? Seems like we should be able to match for top-level sections ending in ` Details` rather than have to configure this through the settings.